goshopify.go

```diff
@@ -234,6 +241,11 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 	}
 
 	if callLimit, ok := resp.Header[callLimitHeader]; ok {
+		// Extract the current bucket capacity from the call limit response header
+		bucketCapacity, err = strconv.Atoi(strings.Split(callLimit[0], "/")[0])
+		if err != nil {
+			return err
```

Comment:
I think you should maybe do the callLimitHeader log before all of this since it is possible to return err.

Reply:
right. but, given Maya's comment it will be logged always